### PR TITLE
WIP: Add optional parameter routePath to WithRoute

### PR DESCRIPTION
**Pull request by liamwazherealso**
WRT to issue https://github.com/openshift/operator-custom-metrics/issues/14

### CODE REVIEW

File: pkg/metrics/builder.go [hunk 1]
```diff
@@ -64,8 +65,15 @@ func (b *metricsConfigBuilder) WithCollectors(collectors []prometheus.Collector)
 	return b
 }
 
-func (b *metricsConfigBuilder) WithRoute() *metricsConfigBuilder {
+func (b *metricsConfigBuilder) WithRoute(routePath string) *metricsConfigBuilder {
 	b.config.withRoute = true
+
+	if routePath == "" {
+		b.config.routePath = routePath
```
Comment (fahlmant) [inline pkg/metrics/builder.go]: I think you have these reversed :)

Comment (liamwazherealso) [reply pkg/metrics/builder.go]: : )

**Comment by fahlmant** [comment]
@liamwazherealso I know the example go file is a bit out of date, but could you make sure that this compiles with no issue?

**Comment by RiRa12621** [comment]
Is this still WIP?

**Comment by fahlmant** [comment]
@liamwazherealso Are you still working on this?

**Comment by liamwazherealso** [comment]
Nope

**Pull request closed without merging**